backend/src/org/jetbrains/kotlinconf/backend/Main.kt

```diff
@@ -33,6 +37,18 @@ fun Application.main() {
     install(WebSockets)
     install(XForwardedHeadersSupport)
     install(StatusPages) {
+        exception<ServiceUnavailable> { _ ->
```

Comment:
Thanks. I applied it previously in Kot. Academy project and this mechanism work really well. Good thing is that now you can throw this errors in any place (like in use cases or in a database) and it will be caught as well.